### ISSUE TITLE
Selecting/Deselecting the first item in values does not announce via screen reader

### ISSUE THREAD

**Issue by ivaynberg** [select2/select2#5568]
First item being the first selected, not the first in the list

**Comment by ivaynberg** [comment]
cannot reproduce with keyboard, happens occasionally when using a mouse. i think it happens because when mouse hovers over elements the screen reader tries to announce a lot and it will drop selected/unselected announcement. dont think there is anything that can be fixed in this case.
